balancer/rls/internal/balancer.go

```diff
@@ -75,18 +76,18 @@ func (lb *rlsBalancer) run() {
 // channel accordingly.
 // TODO(easwars): Handle updates to other fields in the service config.
 func (lb *rlsBalancer) handleClientConnUpdate(ccs *balancer.ClientConnState) {
-	grpclog.Infof("rls: service config: %+v", ccs.BalancerConfig)
+	logger.Infof("rls: service config: %+v", ccs.BalancerConfig)
```

Comment:
Let's remove these "rls:" prefixes everywhere (and similar in other places).  Unless you want to do that as a separate PR.

Reply:
Let's save that for a separate PR. I would expect there to be A LOT of them

Reply:
SGTM